team-config.yml

```diff
@@ -17,6 +17,7 @@ toffee-shared-infrastructure:
 apple:
```

Comment:
did you want apple or toffee or both? I would've thought toffee

Reply:
Thought about doing both but wasn't sure if we wanted to avoid interrupting toffee if it's being used in tests for sds-azure-platform

Reply:
Updated now